src/drivers/github.js

```diff
@@ -290,7 +297,7 @@ class Github {
   }
 
   get branch() {
-    return GITHUB_REF;
+    return branch_name(GITHUB_REF);
```

Comment:
```suggestion
    return GITHUB_REF;
```

It doesn't contain the `refs/heads` prefix, so removing any path components will break on branch names with slashes.

Reply:
@0x2b3bfa0 true. So we have to do it the hard way

Reply:
GITHUB_REF contains refs/heads

Reply:
https://stackoverflow.com/questions/58033366/how-to-get-current-branch-within-github-actions

Reply:
🤦🏼  My bad: tested in the wrong environment, sorry.